src/main/java/org/llorllale/cactoos/matchers/TextMatcher.java

```diff
@@ -37,6 +37,15 @@
  *
  * @since 1.0.0
  * @checkstyle ProtectedMethodInFinalClassCheck (200 lines)
+ * @todo #165:30m Add a constructor to TextMatcher taking a Func<String, Boolean> in order to simplify this TextMatcher:
+ *  new TextMatcher(
+ *     new MatcherOf<>(
+ *         (String act) -> act.endsWith(text.asString()),
+ *             desc -> desc.appendText(text.asString()),
+ *             (actual, desc) -> desc.appendValue(actual)
+ *     ),
+ *     "Text ending with "
+ * )
```

Comment:
@ruca905 I think you need to add one more space in front of this last parenthesis...